eg/eg_exec.py

```diff
@@ -1,10 +1,15 @@
 #!/usr/bin/python
 import argparse
-import eg_config
-import eg_util
 import pydoc
 import sys
 
+try:
+    import eg_config
+    import eg_util
+except ImportError:
+    from eg import eg_config
+    from eg import eg_util
+
```

Comment:
Maybe we should actually be using [explicit dot syntax](https://www.python.org/dev/peps/pep-0328/#guido-s-decision) for relative imports? I didn't know about this, I've yet to look into it in real depth.